docs/install.md

```diff
@@ -191,6 +201,135 @@ To do a debug build, run make like:
 
 If you suspect you'll be hacking on Sandstorm's dependencies as well, you may want to follow the dependency symlink trick described in the Ekam readme.
 
+## Option 5: Integrating with configuration management systems like Ansible/Puppet
+
+Configuration mangement tools like Ansible, Puppet, and Chef allow a sysadmin to declaratively state
+how the server should be configured. This allows sysadmins within an organization to collaborate
+more effectively.
+
+If you want to prepare a server to run Sandstorm using a configuration management system, the
+configuration management system should take the following steps.
+
+- Download install.sh at runtime within the configuration mangement system from
+  [https://install.sandstorm.io/](https://install.sandstorm.io/), and [verify the install.sh
+  signature](#option-3-pgp-verified-install).  Alternatively you can download
+  install.sh into your own trusted file storage area and verify it as part of copying it to your own
+  trusted file storage area.
+
+- Run install.sh with the options of your liking. Examine the [install.sh reference
+  documentation](administering/install-script.md) section for information about non-interactive use
+  of install.sh.
+
+- If you need to make further configuration changes, then stop the Sandstorm service with `sudo
+  service sandstorm stop`, modify the config file in `/opt/sandstorm/sandstorm.conf` so that it
+  contains the contents you want, and start the Sandstorm service.
+
+Note that `BASE_URL`, `WILDCARD_HOST`, and `ALLOW_DEV_ACCOUNTS` are three configuration file options
+whose value you will want to verify. See the [full documentation on
+sandstorm.conf](administering/config-file.md).
+
+You can look at these examples as a starting-point:
+
+- [Sandcastle](https://github.com/iflowfor8hours/sandcastle), an Ansible playbook that installs
+  Sandstorm as part of "An opinionated configuration for running sandstorm with a focus on security
+  and paranoid assumptions."
+
+- [Sandstorm's installer test suite](administering/install-script.md#examples), where you can find some automated
+  invocations of install.sh.
+
+Note that this process uses Sandstorm's install.sh to download Sandstorm. Another option would be if
+Sandstorm provided an APT repository. However, at the time of writing (July 2016), there is no APT
+repository for Sandstorm because we have not yet examined fully how to retain Sandstorm's
+self-containerization and auto-updates in conjunction with an APT repository. If you're interested
+in that feature, please email support@sandstorm.io so we can use that information to prioritize it
+further.
+
+## Option 6: Using Sandstorm within Docker
+
+Docker is a popular tool for declaring how to run code on servers. Sandstorm can run within Docker.
+We suggest running Sandstorm outside of Docker, but if your organization runs all server software
```

Comment:
Proposal:

> We recommend running Sandstorm outside of Docker if you have the option; at the moment, Sandstorm's integration with Docker is somewhat non-idiomatic as you will see below. If your organization runs all server software [in Docker, then go ahead]
or

> We recommend running Sandstorm outside of Docker if you have the option; at the moment, Sandstorm's integration with Docker is somewhat non-idiomatic as you will see below. Most of our Sandstorm testing occurrs outside of Docker, so if you run into problems, please contact us so that we know about the issue. If your organization runs all server software [in Docker, then go ahead]